app/frontend/src/components/JsonBuilder.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <v-data-table
+    class="json-builder"
+    :disable-pagination="true"
+    :headers="headers"
+    :hide-default-header="true"
+    :hide-default-footer="true"
+    :items="items"
+  >
+    <template v-slot:item.key="props">
+      <v-text-field placeholder="Key" single-line v-model="props.item.key" />
+    </template>
+    <template v-slot:item.value="props">
+      <v-text-field placeholder="Value" single-line v-model="props.item.value" />
+    </template>
+    <template v-slot:item.action="{ item }">
+      <v-tooltip top>
+        <template v-slot:activator="{ on }">
+          <v-btn @click="deleteItem(item)" fab class="mb-2" small text v-on="on">
+            <v-icon>delete</v-icon>
+          </v-btn>
+        </template>
+        <span>Add Key/Value Pair</span>
+      </v-tooltip>
+    </template>
+    <template v-slot:footer>
+      <v-container>
+        <v-row justify="end">
```

Comment:
Does this need to be in a row? No responsive columns.

Reply:
This was one of the only succinct ways I could pull the button to the right side of the pane without resorting to some ugly CSS hacks.